manual/command-reference-manual.tex

```diff
@@ -6141,21 +6141,27 @@ \section{synth\_xilinx -- synthesis for Xilinx FPGAs}
         write_blif 
 \end{lstlisting}
 
-\section{tcl -- execute a TCL script file}
```

Comment:
This file is autogenerated from the help text in the C++ file, so your changes need to be done there.